File: src/joyowlsf/baekjoon/_2775.py
T = int(input())

for i in range(T):
    k = int(input())
    n = int(input())
    people_num = [i for i in range(1,n+1)]

    for a in range(k):
        for b in range(1,n):
            people_num[b] += people_num[b-1]
    print(people_num[-1])

File: src/joyowlsf/baekjoon/_2775.java
package joyowlsf.baekjoon;

import java.util.Scanner;

public class _2775 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        int T = sc.nextInt();

        for(int x=0; x<T; x++){
            //k층
            int k = sc.nextInt();
            //n호
            int n = sc.nextInt();
            //사람수
            int people_num[] = new int[n];

            for(int i=1; i<n+1; i++){
                people_num[i-1] = i;
            }
            for(int y=0;y<k;y++){
                for(int j=1;j<n;j++){
                    people_num[j] = people_num[j] + people_num[j-1];
                }
            }
            System.out.println(people_num[people_num.length-1]);
        }
    }
}
